ipmof/energymap.py

```diff
@@ -121,23 +122,44 @@ def energy_map_atom_index(atom_name, emap_atom_list):
     return int(emap_atom_index + 3)
 
 
-def export_energy_map(emap, sim_par, sim_dir, mof_name):
+def export_energy_map(emap, atom_list, sim_par, sim_dir, mof_name):
```

Comment:
does sim_par stand for simulation parameters? if so, a more common abbreviation would be sim_params